homeassistant/components/daikin/climate.py

```diff
@@ -274,22 +274,26 @@ def device_info(self):
     def is_on(self):
         """Return true if on."""
         return self._api.device.represent(
```

Comment:
Does this call return a cached value, ie no I/O?

Reply:
Yes!

Reply:
Just like: https://github.com/home-assistant/home-assistant/pull/23712/files#diff-09fbe0584c521c44cc37d1d27daba4f3R137